### DISCUSSION TITLE
Is there a way to choose a default look&feel

### DISCUSSION THREAD

**Discussion by tilman-schieber** [LiaScript/LiaScript#203] [Q&A] [answered]
Is there a way to add a default "code theme" or "color theme" to a liascript file? So that it starts in dark-mode, or with a certain syntax highlighting theme enabled?

**Comment by andre-dietrich** [comment, marked as answer]
You could do this via accessing `window.LIA.settings`. All settings there mirror the user settings and can be inspected and modified ... however the user can change them back ;-)

```` markdown
<!--
dark: true

@onload
window.LIA.settings.editor = "dreamweaver"
window.LIA.settings.theme = "red"
@end
-->

# My Custom Settings

```javascript
// all settings are also accessable via the LIA object
console.log(window.LIA.settings)

// change the editor theme
window.LIA.settings.editor = "ambiance"
```
<script>@input</script>
````

[Live Example](https://liascript.github.io/LiveEditor/?/show/code/H4sIAAAAAAAAA4WPv27CQAzGdz+FSRcYkuxVQEGdKsHUF4hzZ4WDyxndOYn69k1o2JAYbX1/fl+1yXOwFG+fqHFggFqCF7IwuWBlKk7fxyKxqgtdKtg6lYh7zGxk6iemkWP2UqoX7nlRRrYZ1Bws5PkB4APPv/g1JJUef1YxQNM0VxopmejuCmWJ5D0+s5Aiz48kSMZwStR6xtERzh04l6K0VzYKRkISz4WXbvsCaQdLsLlQ6PhhXdc8SN/Mpb51FAxnCylU/5yH2oX7oFW5nvAH+03E7EwBAAA=)

**Reply by tilman-schieber** [reply]
Yes perfect! it's fine the user can change it back, I just want to suggest a nice default theme...
